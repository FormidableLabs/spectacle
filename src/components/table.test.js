import React from 'react';
import { mount } from 'enzyme';
import Table from './table';

describe('<Table />', () => {
  test('should render correctly', () => {
    const context = { styles: { components: { table: {} } } };
    const wrapper = mount(
      <Table>
        <tbody>
          <tr>
            <td>Table Content</td>
          </tr>
        </tbody>
      </Table>,
      { context }
    );
    expect(wrapper).toMatchSnapshot();
  });
});
