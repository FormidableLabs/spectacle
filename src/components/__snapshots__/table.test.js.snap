// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Table /> should render correctly 1`] = `
<Table>
  <Styled(table)
    className={undefined}
    styles={
      Array [
        Object {},
        Object {},
        undefined,
      ]
    }
  >
    <table
      className="css-0"
    >
      <tbody>
        <tr>
          <td>
            Table Content
          </td>
        </tr>
      </tbody>
    </table>
  </Styled(table)>
</Table>
`;
